Uchu.StandardScripts/Equipment/Coil.cs

```diff
@@ -0,0 +1,69 @@
+using System.Threading.Tasks;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.Equipment
+{
+    public class Coil : ObjectScript
```

Comment:
Does this need Annotations?

Reply:
Don't think so because those are on CoilBackpack, GemPack and ShardArmor, which are the scripts that are actually attached to objects and they inherit from this class